apps/web/src/app/[locale]/shop/SellListSection/EditModal.tsx

```
@@ -50,24 +52,26 @@ function EditModal({ isOpen, onClose, productId }: { isOpen: boolean; onClose: (
-        </Button>
-      </div>
-    </Modal>
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <Dialog.Content>
+        <Dialog.Title className={titleStyle}>{t('edit-product')}</Dialog.Title>
+        <input
+          className={inputStyle}
+          placeholder="Type price..."
+          type="number"
+          value={Boolean(price) ? price : ''}
+          onChange={(e) => setPrice(Number(e.target.value))}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)